subcommands/keys/tuf_updates_add_offline_key.go

```diff
@@ -0,0 +1,124 @@
+package keys
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	tuf "github.com/theupdateframework/notary/tuf/data"
+
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
+)
+
+func init() {
+	add := &cobra.Command{
+		Use:   "add-offline-key --role root|targets --txid=<txid> --keys=<tuf-root-keys.tgz>",
+		Short: "Stage addition of the offline TUF signing key for the Factory",
+		Long: `Stage addition of the offline TUF signing key for the Factory.
+
+The new offline signing key will be used in both CI and production TUF root.
+
+When you add a new TUF targets offline signing key, existing production targets are not signed by it.
+Please, use the sign-prod-targets subcommand if you want to sign existing production targets with a new key.`,
```

Comment:
>So, together these problems necessitate to add a separate command to sign prod targets.

makes sense to me.

Reply:
The aforementioned changes require changes to the API, so that the user can view which production signatures were added to the transaction.
So, I'll do them as a separate PR.

Reply:
https://github.com/foundriesio/fioctl/pull/244